bin/generate-dummy-tasks

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env node
+try {
+  // eslint-disable-next-line implicit-dependencies/no-implicit
+  require('dotenv').config();
+} catch (e) {}
+
+if (process.env.NODE_ENV !== 'development') {
+  console.log('NON-DEV ENV DETECTED - THIS SCRIPT WIPES ALL TASK DATA, EXITING');
```

Comment:
This is temporary - will either delete script or exclude from deployment before merge.